index.html

```diff
@@ -75,7 +48,59 @@ <h1>Active Flights</h1>
           ></path>
         </svg>
       </div>
+      <div id="loading-message" class=" bg-primary-bg-color text-center"></div>
+    </div>
+    <div id="container" class="flex h-full flex-col content-start">
+      <header class="heading m-0 flex h-fit bg-primary-bg-color p-0">
+        <div class="logo h-44 w-44"></div>
+        <div
+          class="flight-track grid w-auto flex-grow grid-cols-[3rem,4rem,3.5rem,auto] grid-rows-2 gap-0 pr-10 md:pr-20"
+        >
+          <p
+            class="flight-heading col-span-4 m-0 self-end border-b-4 text-4xl font-semibold"
+          >
+            FLIGHT
+          </p>
+          <p class="track-heading col-start-2 m-0 text-4xl font-semibold">
+            TRACK
+          </p>
+        </div>
+      </header>
+      <div
+        class="content flex h-full w-full flex-col md:text-[1.1rem] lg:text-[1.2rem]"
+      >
+        <main class="active-flights flex h-fit flex-col justify-center p-4">
+          <div class="active-flights-heading flex">
+            <h1 class="font- m-0 self-start text-2xl">Active Flights</h1>
+            <span class="self-end pl-1 pb-1 text-xs"
+              >(Over South-African airspace)</span
+            >
+          </div>
+          <div class="flights flex flex-col pt-4">
+            <div
+              class="flight-headings ml-2 mr-2 grid grid-cols-4 pb-1 text-[0.7rem] font-medium md:grid-cols-5 md:text-[0.8rem] lg:grid-cols-6 lg:text-[0.9rem] xl:grid-cols-7"
```

Comment:
`ml-2 mr-2 -> mx-2`

Reply:
personally not the biggest fan of the custom text classes `text-[0.9rem]` it bulks up bundle size since tailwind has to create those.... maybe use the builtin classes but also up to you. For this project it doesn't matter much but in bigger ones it does cause issues

Reply:
This was used as a direct translation from scss to tailwind , but I will keep it in mind for the next project.

mx-2 added